pkgs/desktops/gnome-2/desktop/gopenvpn/default.nix

```diff
@@ -0,0 +1,62 @@
+{ stdenv, fetchurl, fetchFromGitHub, pkgconfig, glib, gtk2, gnome2, dbus_glib, gmime, libnotify, libgnome_keyring,
+  openssl, cyrus_sasl, gnonlin, sylpheed, gob2, gettext, intltool, libglade, polkit, autoconf, automake, expect,
+  openvpn, makeWrapper, pkexecPath ? "/var/setuid-wrappers/pkexec" }:
+
+let
+  redirects = [ "/usr/bin/pkexec=${pkexecPath}" ];
+in stdenv.mkDerivation rec {
+  rev = "2b9285f1ea2d11434b8ee8e10d937b2ea5285b63";
+  version = "0.8";
+  name = "gopenvpn-${version}";
+
+  src = fetchFromGitHub {
+    inherit rev;
+    owner = "proteansec";
+    repo = "gopenvpn";
+    sha256 = "0pkdravr5wm7phdr2rd1rx1ry4dpwkrf0hv7wi99n4ywsl5jqwq8";
+  };
+
+  nativeBuildInputs = [ pkgconfig ];
+  buildInputs = [ glib gtk2 dbus_glib gmime libnotify libgnome_keyring openssl cyrus_sasl gnonlin sylpheed gob2
+                  gettext intltool libglade polkit pkgconfig autoconf automake expect openvpn makeWrapper ];
```

Comment:
By adding the _autoreconfHook_ the program does not compile and results in the following error message:

```
autoreconf: running: aclocal --force -I m4
autoreconf: configure.ac: tracing
autoreconf: configure.ac: not using Libtool
autoreconf: running: /nix/store/rdnli6k94jlgjpyf8ccr9hgdp3pi4p01-autoconf-2.69/bin/autoconf --force
autoreconf: running: /nix/store/rdnli6k94jlgjpyf8ccr9hgdp3pi4p01-autoconf-2.69/bin/autoheader --force
autoreconf: running: automake --add-missing --copy --force-missing
Unescaped left brace in regex is deprecated, passed through in regex; marked by <-- HERE in m/\${ <-- HERE ([^ \t=:+{}]+)}/ at /nix/store/483rb2q3h37xkwr8fjrsk6bz59x5ypg2-automake-1.15/bin/automake line 3936.
automake: warnings are treated as errors
configure.ac:25: warning: The 'AM_PROG_MKDIR_P' macro is deprecated, and its use is discouraged.
configure.ac:25: You should use the Autoconf-provided 'AC_PROG_MKDIR_P' macro instead,
configure.ac:25: and use '$(MKDIR_P)' instead of '$(mkdir_p)'in your Makefile.am files.
configure.ac:25: installing './compile'
configure.ac:25: installing './config.guess'
configure.ac:25: installing './config.sub'
configure.ac:23: installing './install-sh'
configure.ac:23: installing './missing'
src/Makefile.am: installing './depcomp'
autoreconf: automake failed with exit status: 1
builder for ‘/nix/store/1p6iswcb2jg5wn6466nvg9i0ksshmjld-gopenvpn-0.8.drv’ failed with exit code 1
```

Even with the current code in place, the program compiles only by issuing the following commands one after another:
```
# autopoint -f
# gettextize -f
# autoreconf -vif 
```